src/actions/index.js

```diff
@@ -0,0 +1,150 @@
+import axios from 'axios';
+import jwt from 'jsonwebtoken';
+
+import instance from './AxiosInstance';
+import { ROOT_URL } from '../constants';
+
+export const AUTHENTICATED = 'authenticated_user';
+export const UNAUTHENTICATED = 'unauthenticated_user';
+export const AUTHENTICATED_ERROR = 'authenticated_error';
+export const REGISTER = 'register_user';
+export const USER_ALREADY_EXISTS = 'user_already_exists';
+export const FETCH_CATEGORIES = 'fetch_categories';
+export const FETCH_RECIPES = 'fetch_recipes';
+export const CREATE_CATEGORY = 'create_category';
+
+// action creator for signing in user
+export const login = (res) => {
+    return {
+        type: AUTHENTICATED, 
+        user: jwt.decode(res.data.auth_token),
+        payload: res
+    }
+}
```

Comment:
Where is this login function used? It looks like you opted to dispatch the login manually in the signIn action below